src/CommonDCModelDropdown.php

```diff
@@ -438,6 +438,45 @@ public function displaySpecificTypeField($ID, $field = [], array $options = [])
         }
     }
 
+
+    public static function rawSearchOptionsToAdd()
+    {
+        global $DB;
+
+        $soptions = [];
+
+        $soptions[] = [
+            'id'   => 'pictures',
+            'name' => _n('Picture', 'Pictures', Session::getPluralNumber())
+        ];
+
+        $soptions[] = [
+            'id'                 => '250',
+            'table'              => self::getTable(),
+            'field'              => 'picture_front',
+            'name'               => __('Front picture'),
+            'datatype'           => 'specific',
+            'nosearch'           => true,
+            'nosort'           => true,
+            'massiveaction'      => false,
+            'html'               => true
```

Comment:
`$options['html']` tested in `getSpecificValueToDisplay` is not coming from here, you can remove it.
```suggestion
```